README.md

```diff
@@ -388,4 +388,4 @@ Many thanks for valuable feedback and advice from:
 - Phil Smart
 - Martin Thompson
 - Christian Biesinger
-- Erica Kovac
\ No newline at end of file
+- Erica Kovac 
```

Comment:
I think the github web ui is adding an extra new line at the end of this file, and hence creating this diff ...

Reply:
Yep, there's a similar diff on my other PR.